TheraLang.Web/Controllers/MemberFeeController.cs

```diff
@@ -2,13 +2,15 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
 using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using TheraLang.BLL.DataTransferObjects;
 using TheraLang.BLL.Interfaces;
 using TheraLang.Web.ViewModels;
 
 namespace TheraLang.Web.Controllers
 {
+    [Authorize]
```

Comment:
As I know this controller is accessible only for admin, maybe you should use Roles = RolesConstants.Admin?

Reply:
Thanks)))Done